server/data/visitSchedulerApiClient.ts

```diff
@@ -217,6 +218,13 @@ export default class VisitSchedulerApiClient {
     })
   }
 
+  async updateLocationGroup(reference: string, updateLocationGroupDto: UpdateLocationGroupDto): Promise<LocationGroup> {
+    return this.restClient.post({
```

Comment:
```suggestion
    return this.restClient.put({
```
According to Swagger (https://visit-scheduler-dev.prison.service.justice.gov.uk/swagger-ui/index.html#/7.%20Location%20group%20admin%20rest%20controller/updateLocationGroup) this should be a `PUT` request?